### PR TITLE
add simple mobile tools pro, fix drawables for simple mobile tools

### PR DESCRIPTION
**Pull request by swedneck**
Looks like all the simple mobile tools apps were using "simplecalculator" for the drawable, whoops :P

### CODE REVIEW

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -1121,247 +1121,487 @@
...
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Amber}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Orange}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Deep_orange}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Brown}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Blue_grey}" drawable="simplecalculator"/>
-    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Grey_black}" drawable="simplecalculator"/>
+    <item component="ComponentInfo{com.simplemobiletools.calculator/com.simplemobiletools.calculator.activities.SplashActivity.Pink}" drawable="simplecamera"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]: shouldn't these use `simplecalculator` instead of `simplecamera`

Comment (swedneck) [reply app/src/main/res/xml/appfilter.xml]: They should, vim got a bit overzealous with the search and replace.

**Comment by swedneck** [comment]
Is this ok to merge now?

**Comment by dkanada** [comment]
Yeah I am just a bit behind on merge requests right now. Yours didn't add any new icons though, so I just merged it.

**Pull request merged**